src/renderers/WebGLRenderTarget.js

```diff
@@ -88,7 +88,10 @@ class WebGLRenderTarget extends EventDispatcher {
 
 		this.viewport.copy( source.viewport );
 
+		// By default texture.clone retains a reference to the original image data which
+		// isn't correct for WebGLRenderTarget. See issue #20328.
 		this.texture = source.texture.clone();
+		this.texture.image = { ...this.texture.image };
```

Comment:
Do you mind just adding the comment like so:
```js
this.texture.image = { ...this.texture.image }; // see #20328
```

Reply:
Done!